client/me/concierge/cancel/index.js

```diff
@@ -98,7 +98,7 @@ class ConciergeCancel extends Component {
 							description={ translate(
 								'You can also reschedule your session. What would you like to do?'
 							) }
-							title={ translate( 'Cancel your Concierge session' ) }
+							title={ translate( 'Cancel your session' ) }
```

Comment:
:disappointed: 19 existing translations will be lost with this change.
If the string must change, the following option has already been translated **19** times:
`translate( 'Cancel session' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).